go/lang/security/audit/crypto/math_random.yaml

```diff
@@ -5,16 +5,31 @@ rules:
     owasp: 'A3: Sensitive Data Exposure'
     references:
     - https://cheatsheetseries.owasp.org/cheatsheets/Cryptographic_Storage_Cheat_Sheet.html#secure-random-number-generation
-  message: Do not use math.rand. Use `crypto/rand` instead.
+  message: Do not use `math/rand`. Use `crypto/rand` instead.
   languages: [go]
   severity: WARNING
-  patterns:
-  - pattern-either:
+  pattern-either:
+  - pattern: |
+      import mrand "math/rand"
```

Comment:
Could these patterns be simplified using a `pattern-inside` clause with the below?

```
import mrand "math/rand"
...
```

Reply:
Yup! Good catch, that improves the `ruleid` location :+1: 

Updating...